index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family='Poppins':wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>My mobile portfolio</title>
+  </head>
+  <body>
+    <main>
+      <nav class="portfolio-nav">
+        <h2 class="portfolio-nav--logo">My logo</h2>
+        <ul class="portfolio-nav--nav-options">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
+      </nav>
+      <section class="headline">
+        <div class="headline--info">
+          <h1 class="headline--info--name">
+            I’m Nahom Zerihun Glad to see you!
+          </h1>
+          <p class="headline--info--desc">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3>LET'S CONNECT</h3>
+        </div>
+
+        <ul class="headline--social-media">
+          <li>
+            <a href=""><img src="images/twitter.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="images/Linkedin icon.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="images/medium.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="images/github.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="images/angellist.png" alt="" /></a>
```

Comment:
Please make the `alt` attribute value more descriptive instead of an empty string because it will increase your website's accessibility (When a user is using a screen reader it read the alt value whenever it encounters an `img` tag check [this](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) for more info)